public/js/action-list.js

```diff
@@ -799,6 +799,10 @@
 
                 _this.clearSelection(_this.getAllItems(list).filter(item => !toActiveItems.includes(item)));
                 _this.setActive(toActiveItems);
+
+                if (toActiveItems.length) {
+                    _this.scrollItemIntoView(toActiveItems[toActiveItems.length - 1], 'ArrowDown');
+                }
```

Comment:
Please undo this. It's bad practice to perform such big changes such as scrolling without the user's input.